string name
float32 blockSize
float32 visionScope
---
bool found